FreeRTOS/FreeRTOS
# Add unit tests for stream buffer callback overrides

Add unit tests for stream buffer callback overrides (PR https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/437)

Description
-----------

1. Adds unit tests for new APIs  to override stream buffer and message buffer callbacks.
2. Updated some of the existing tests to handle corner cases.
3. Updates kernel Submodule

Test Steps
-----------
None

Related Issue
-----------

https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/437

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
